doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -392,7 +435,7 @@ To configure the Hash algorithms, set one or more of the following Kconfig optio
 +-----------------------+---------------------------------------------------+
 | Hash algorithm        | Configuration option                              |
 +=======================+===================================================+
-| SHA-1                 | :kconfig:option:`CONFIG_PSA_WANT_ALG_SHA_1`       |
+| SHA-1 (unsafe)        | :kconfig:option:`CONFIG_PSA_WANT_ALG_SHA_1`       |
```

Comment:
Can we use "weak" instead of "unsafe"? Mbed TLS has that as an established term

Reply:
Sure, that's a valid point, especially since it's followed by NIST.